[ALL] Completamento glossario (#137)
Argo-swe/Docs: sources/documents/2_RTB/Glossario/Lettere/c.tex

\vspace{2em}
\subsection*{ChatBOT}
TODO

\vspace{2em}
\subsection*{ChatGPT}
TODO

\vspace{2em}
\subsection*{Chinook}
TODO

\vspace{2em}
\subsection*{Codifica}
TODO

\vspace{2em}
\subsection*{Commit}
Un commit è un'operazione in un sistema di controllo versione che salva le modifiche apportate al codice sorgente o ai documenti in un repository. Rappresenta una "istantanea" dello stato del progetto in un determinato momento, accompagnata da un messaggio che descrive le modifiche effettuate. Il commit permette di tracciare la storia delle modifiche, facilitando il lavoro collaborativo e il mantenimento del codice, consentendo di tornare a versioni precedenti se necessario.

\vspace{2em}
\subsection*{Committente}
TODO

\vspace{2em}
\subsection*{Containerizzazione}
TODO

\vspace{2em}
\subsection*{Continuous Integration}
TODO

\vspace{2em}
\subsection*{CRUD}

> Acronimo che fa riferimento alle 4 operazioni base relative alla gestionre della persistenza dei dati. Le operazioni sono Create (creazione), Read (lettura), Update (aggiornamento), Delete (cancellazione).

Reviewer: ```suggestion
Acronimo che fa riferimento alle 4 operazioni base relative alla gestione della persistenza dei dati. Le operazioni sono Create (creazione), Read (lettura), Update (aggiornamento), Delete (cancellazione).
```

\vspace{2em}
\subsection*{CRUDL}

> \par Con riferimento a \glossario{Large Language Model} aggiunge l'operazione List, per la rapresentazione in formato lista di risorse multiple.

Reviewer: ```suggestion
\par Con riferimento a \glossario{Large Language Models} aggiunge l'operazione List, per la rappresentazione in forma di lista di risorse multiple.
```

\vspace{2em}
\subsection*{CSV}

> Formato di file basato su righe di testo in cui i valori vengono separati da virgole. Viene utilizzato principalmente per l'importazione ed esportazione dei dati. La struttura semplice non permetta la gestione di modelli di dati complessi.

Reviewer: ```suggestion
Formato di file basato su righe di testo in cui i valori vengono separati da virgole. Viene utilizzato principalmente per l'importazione ed esportazione dei dati. La struttura semplice non permette la gestione di modelli di dati complessi.
```

> Acronimo che fa riferimento alle 4 operazioni base relative alla gestione della persistenza dei dati. Le operazioni sono Create (creazione), Read (lettura), Update (aggiornamento), Delete (cancellazione).

Reviewer 2: ```suggestion
Acronimo che fa riferimento alle 4 operazioni base relative alla gestione della persistenza dei dati. Le operazioni sono: Create (creazione), Read (lettura), Update (aggiornamento), Delete (cancellazione).
```
